server/services/azureService.js

```diff
@@ -417,49 +457,32 @@ const combineAllChunksInToOneVideo = async (userId, jobId, questionId) => {
     );
   }
 
-  await uploadFinalVideoToAzure(
-    combinedVideoPath,
-    `${userId}/${jobId}/${questionId}/${combinedVideoName}`
-  );
-
-  await deleteChunksFromAzure(chunks);
+  console.log(combinedVideoPath);
 
-  await new Promise((resolve) => setTimeout(resolve, 2000));
+  await extractAudio(combinedVideoPath, audioPath);
 
-  chunkPaths.forEach((chunkPath) => {
-    try {
-      fs.unlinkSync(chunkPath);
-    } catch (err) {
-      console.error(`Error deleting chunk ${chunkPath}:`, err);
-    }
-  });
+  const transcription = await transcribeAudio(audioPath);
 
-  const questionDir = path.join(
-    __dirname,
-    "tempChunks",
+  await InterviewService.updateAnswer(
     userId,
     jobId,
-    questionId
+    questionId,
+    transcription.toString().trim()
   );
-  const jobDir = path.join(__dirname, "tempChunks", userId, jobId);
+  console.log("Updated answer in DB successfully");
 
-  try {
-    fs.rmSync(questionDir, { recursive: true, force: true });
-  } catch (err) {
-    console.error(`Error deleting questionId directory ${questionDir}:`, err);
-  }
+  // await uploadFinalVideoToAzure(
```

Comment:
Why is this commented?

Reply:
While testing I commented it forgot to uncomment